src/nmmodel.cpp

```diff
@@ -170,7 +170,7 @@ void NmModelPrivate::clearConnections()
         removeConnection(0);
 }
 
-void NmModelPrivate::addConnection(NetworkManager::Connection::Ptr conn)
+void NmModelPrivate::addConnection(NetworkManager::Connection::Ptr &conn)
```

Comment:
Please, use the notation/style as in other places => `&` belongs to type... not to variable name.